astroquery/ned/core.py

```diff
@@ -718,7 +718,7 @@ def _parse_radius(radius):
         The value of the radius in arcminutes.
 
     """
-    return coord.Angle(radius).to('arcmin').value
+    return coord.Angle(radius).arcmin
```

Comment:
@AyushYadav - Could you remove these functions (both from here, and from nvas)? Obviously then you can remove their tests from the test files, too so nothing will fail in there either.